src/common/ceph_crypto.cc

```diff
@@ -78,4 +86,45 @@ ceph::crypto::HMAC::~HMAC()
 
 #else
 # error "No supported crypto implementation found."
-#endif
+#endif /*USE_NSS*/
+
+#ifdef USE_OPENSSL
+
+const EVP_MD *SECOidTag_to_EVP_MD(SECOidTag _type) {
+  switch (_type) {
+  case SEC_OID_MD5:
+    return EVP_md5();
+  case SEC_OID_SHA1:
+    return EVP_sha1();
+  case SEC_OID_SHA256:
+    return EVP_sha256();
+  default:
+    return NULL;
```

Comment:
`nullptr` would be preferred.

Reply:
No longer needed because of another change.